sdk/core/test/integration/services/WalletStorageService.test.ts

```diff
@@ -79,7 +84,11 @@ describe('WalletStorageService', () => {
   it.skip('#storeEncryptedSeed throws exception WHEN userId does not exists', async () => {
     const { accessToken } = await userManagementService.logInWithPassword(nonExistingUser, cognitoPassword)
 
-    const keyStorageApiService = new KeyStorageApiService(options)
+    const keyStorageApiService = new KeyStorageApiService({
+      keyStorageUrl: options.keyStorageUrl,
+      accessApiKey: options.accessApiKey,
+      sdkVersion: extractSDKVersion(),
+    })
```

Comment:
Can't this be moved to `beforeEach`? They look exactly the same and now it's more visible that it's just a repeated code.

Reply:
That does not needed for every test in suite to put it into `beforeEach` and don't want that each test could use same instance of `KeyStorageApiService` service created in `beforeAll`. But we actually can move service creation into separate function which removes repeated lines.